server/controllers/personController.js

```diff
@@ -0,0 +1,69 @@
+var Person = require('../models/Person');
+
+exports.index = function(req, res) {
+    Person.find({})
+    .catch((err) => {
+        res.status(400).send(err);
+    })
+    .then((result) => {
+        res.status(200).json(result);
+    });
+};
+
+exports.show = function(req, res) {
+    Person.findById(req.params.person_id)
+    .catch((err) => {
+        res.status(400).send(err);
+    })
+    .then((result) => {
+        res.status(200).json(result);
+    });
+};
+
+exports.create = function(req, res) {
+    var person = new Person(req.body);
+
+    person.save()
+    .catch((err) => {
+        res.status(400).send(err);
+    })
+    .then((createdPerson) => {
+        res_json = {
+            "message": "Person created",
+            "data": {
+                "personId": createdPerson._id,
+            }            
+        }
+        res.status(200).send(res_json);
```

Comment:
Se for enviar json na response, acredito que usar res.status(xxx).**json**() seja a convenção (em vez de .send()).